README.md

```diff
@@ -1,2 +1,44 @@
 # comp110-worksheet-B
 Base repository for COMP110 worksheet B
+
+```
+enter terminal
```

Comment:
The worksheet asked just for the pseudocode for the likeness calculation -- you have done more work than needed!